core/lib/zksync_core/src/consensus/tests.rs

```diff
@@ -14,15 +14,26 @@ use zksync_consensus_roles::{node, validator::testonly::Setup};
 use zksync_consensus_storage as storage;
 use zksync_consensus_storage::PersistentBlockStore as _;
 use zksync_protobuf_config::testonly::{encode_decode, FmtConv};
+use zksync_types::{L1BatchNumber, MiniblockNumber};
 
 use super::*;
+use crate::utils::testonly::Snapshot;
+
+async fn new_store(from_snapshot: bool) -> Store {
+    match from_snapshot {
```

Comment:
Bikeshedding: Having a `match` on a `bool` value rubs me the wrong way, but that's completely subjective.

Reply:
bool is an inductive type, and I don't see a reason why not treat it as such.